samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -574,26 +596,25 @@ mfxStatus CEncodingPipeline::InitMfxEncParams(sInputParams *pInParams)
 
     // configure and attach external parameters
     if (MVC_ENABLED & pInParams->MVC_flags)
-        m_EncExtParams.push_back((mfxExtBuffer *)&m_MVCSeqDesc);
+    {
+        auto seqBuffer = m_mfxEncParams.AddExtBuffer<mfxExtMVCSeqDesc>();
```

Comment:
not necessarily to define "auto seqBuffer" here.

Reply:
Removed